syncer/sharding_group.go

```diff
@@ -557,9 +542,11 @@ func (k *ShardingGroupKeeper) UnresolvedTables() (map[string]bool, [][]string) {
 	tables := make([][]string, 0, 10)
 	k.RLock()
 	defer k.RUnlock()
+	// TODO: where does k.groups delete its key?
```

Comment:
Can we delete the key after all tables/schemas leave the group (the group becomes empty)?